package/MDAnalysis/analysis/nuclinfo.py

```diff
@@ -763,3 +770,80 @@ def pseudo_dihe_baseflip(universe, bp1, bp2, i,
     pseudo = mdamath.dihedral(x[0] - x[1], x[1] - x[2], x[2] - x[3])
     pseudo = np.rad2deg(pseudo) % 360
     return pseudo
+
+
+def angle_between_base_planes(universe, b1, b2, seg1="SYSTEM", seg2="SYSTEM"):
+    """The angle between the planes for given two bases is computed.
+
+    See [Gabb1996]_ and [Jafilan2012]_ for background information.
+
+    .. Note:: This angle calculation will only work if using atom names as
+              documented by charmm force field parameters.
+
+    Parameters
+    ----------
+    universe : Universe
+        :class:`~MDAnalysis.core.universe.Universe` containing the
+        trajectory
+    b1 : int
+        resid of the first base
+    b2 : int
+        resid of the second base
+    segid1 : str (optional)
+        segid of b1
+    segid2 : str (optional)
+        segid of b2
+
+    Returns
+    -------
+    float
+          angle in degrees
+
+
+    .. versionadded:: 2.1.0
+    """
+    batms = {'CYT': ['N1', 'N3', 'C5'],
+             'THY': ['N1', 'N3', 'C5'],
+             'URA': ['N1', 'N3', 'C5'],
+             'ADE': ['N9', 'N1', 'N7'],
+             'GUA': ['N9', 'N1', 'N7'],
+             'C': ['N1', 'N3', 'C5'],
+             'T': ['N1', 'N3', 'C5'],
+             'U': ['N1', 'N3', 'C5'],
+             'A': ['N9', 'N1', 'N7'],
+             'G': ['N9', 'N1', 'N7']}
+    # select residues
+    bf1 = universe.select_atoms(
+        "(segid {0!s} and resid {1!s})".format(seg1, b1))
+    bf2 = universe.select_atoms(
+        "(segid {0!s} and resid {1!s})".format(seg2, b2))
+    for ag, segid, res in [(bf1, seg1, b1), (bf2, seg2, b2)]:
+        if len(ag.residues) > 1:
+            warnings.warn(f"Found more than one residues with resid {res} and "
+                          f"segid {segid}. Using first residue in selection.")
+    # get residue names
+    resn1 = bf1.atoms.resnames[0]
+    resn2 = bf2.atoms.resnames[0]
+    # extract positions for specific atoms of each base
+    try:
+        c11 = bf1.select_atoms("name {0!s}".format(batms[resn1][0])).positions[0]
+        c12 = bf1.select_atoms("name {0!s}".format(batms[resn1][1])).positions[0]
+        c13 = bf1.select_atoms("name {0!s}".format(batms[resn1][2])).positions[0]
+        c21 = bf2.select_atoms("name {0!s}".format(batms[resn2][0])).positions[0]
+        c22 = bf2.select_atoms("name {0!s}".format(batms[resn2][1])).positions[0]
+        c23 = bf2.select_atoms("name {0!s}".format(batms[resn2][2])).positions[0]
+    except KeyError:
+        errmsg = ("Residue names may be incorrect! "
+                  "Please use either ADE/GUA/CYT/URA/THY or A/G/C/U/T format.")
+        raise KeyError(errmsg)
+    except IndexError:
+        errmsg = ("Atom names may be incorrect! "
+                  "Make sure base atom names follow CHARMM format.")
+        raise IndexError(errmsg)
+    # get normals to the planes of the bases
+    n1 = mdamath.normal(c12-c11, c13-c11)
```

Comment:
This isn't PBC aware and probably should be. See minimize_vectors which recently got added to lib.

Reply:
good point. Usually users recenter the trajectories before analysis and this may not be always necessary. So do you think in this case the box input to minimize_vectors should be optional or not?